README.md

```diff
@@ -93,37 +129,19 @@ Run `foundry version` to verify it's installed correctly.
     * `Check_if_Registry_key_Value_Exist.yml`: Workflow to invoke the `Check_Registry_Exist` RTR script against a collection of hosts.  Results are written to LogScale.
     * `Notify_status.yml`: Workflow which notifies the `job_history` function to report results of the `Check_if_files_or_registry_key_exist` and `Check_if_Registry_key_Value_Exist.yml`.
 
-## Running, deploying and installing the app
-
-For detailed info about running, deploying and installing this app in your CID, see the Falcon Foundry product documentation:
-
-* Overview and setup
-    * US-1: [Before you begin](https://falcon.crowdstrike.com/documentation/page/f5f7cd69/falcon-console-user-interface-capabilities)
-    * US-2: [Before you begin](https://falcon.us-2.crowdstrike.com/documentation/page/f5f7cd69/falcon-console-user-interface-capabilities)
-    * EU-1: [Before you begin](https://falcon.eu-1.crowdstrike.com/documentation/page/f5f7cd69/falcon-console-user-interface-capabilities)
-* Deploy an app
-    * US-1: [Deploy an app](https://falcon.crowdstrike.com/documentation/page/ofd46a1c/deploy-an-app)
-    * US-2: [Deploy an app](https://falcon.us-2.crowdstrike.com/documentation/page/ofd46a1c/deploy-an-app)
-    * EU-1: [Deploy an app](https://falcon.eu-1.crowdstrike.com/documentation/page/ofd46a1c/deploy-an-app)
-* Create a new app using this app as template
-    * US-1: [Create an app from a template](https://falcon.crowdstrike.com/documentation/page/l159717b/create-an-app#c4378b86)
-    * US-2: [Create an app from a template](https://falcon.us-2.crowdstrike.com/documentation/page/l159717b/create-an-app#c4378b86)
-    * EU-1: [Create an app from a template](https://falcon.eu-1.crowdstrike.com/documentation/page/l159717b/create-an-app#c4378b86)
-* Run this app in development mode after deployment
-    * US-1: [Iterate in development mode](https://falcon.crowdstrike.com/documentation/page/fb88e442/view-and-manage-apps#d5175ae2)
-    * US-2: [Iterate in development mode](https://falcon.us-2.crowdstrike.com/documentation/page/fb88e442/view-and-manage-apps#d5175ae2)
-    * EU-1: [Iterate in development mode](https://falcon.eu-1.crowdstrike.com/documentation/page/fb88e442/view-and-manage-apps#d5175ae2)
-* Work with the Foundry capabilities of this app
-    * US-1: [App capabilities](https://falcon.crowdstrike.com/documentation/category/u0daabab/app-capabilities)
-    * US-2: [App capabilities](https://falcon.us-2.crowdstrike.com/documentation/category/u0daabab/app-capabilities)
-    * EU-1: [App capabilities](https://falcon.eu-1.crowdstrike.com/documentation/category/u0daabab/app-capabilities)
+## Running, deploying and installing Custom Apps
```

Comment:
What do you think about focusing this section on sample apps, rather than custom apps? In which case we could keep the final bullet, link to the editing apps docs (either https://falcon.crowdstrike.com/documentation/page/fb88e442/view-and-manage-apps or https://falcon.crowdstrike.com/documentation/page/fb88e442/view-and-manage-apps#sb0b98b4 - there's not really a good spot to link to), and maybe mention the App templates page - and remove the other general links.

Reply:
There are a couple of things I didn't like about this section before I changed it:

1. The links have the same text over and over for the different environments. This didn't look great IMO, so I trimmed it down to just the environments. I wish there was only one link for each resource rather than having to link to each environment, but I'm not sure this is something we can change. 
2. The intro says "detailed info about running, deploying and installing this app in your CID" when the links go to generic documentation about deploy an app, not **this** app. That's why I changed it from "this app" to "custom apps". 

To be honest, I'm not sure this section is helpful because most folks will probably want to just get the app deployed rather than wander through documentation. A CTA (Call to Action) after getting the app deployed might be more useful.

Reply:
agreed on the overall lack of helpfulness of this section - we don't have much documentation on sample apps specifically, and linking to general docs is likely confusing.

And what you've done with reducing the text to the environment names is also what Support does when sending release note emails, so I think that technique works well. Unfortunately I think providing all 3 links is mandatory - so let's reduce the noise by just providing 1 or 2 targeted links, and then the link to the Foundry doc category.

Reply:
I changed it to show three links, but would be happy to reduce it to two.

Reply:
After some sleep, I'm inclined to ditch this entire section.

Reply:
yes, thanks for confirming!